src/Tasks/ResolveSDKReference.cs

```diff
@@ -61,11 +61,6 @@ public class ResolveSDKReference : TaskExtension
         /// </summary>
         private const string SDKPlatformVersion = "PlatformVersion";
 
-        /// <summary>
-        /// Split char for strings
-        /// </summary>
-        private static readonly char[] s_semicolonSplitChar = { ';' };
```

Comment:
Is there a distinction between this, where you removed the private name, versus others where you kept the private name but pointed it to the `MSBuildConstants` one?

Reply:
Typically I left the existing name alone when its name has more semantic meaning locally or a good doc comment. Example from XMake.cs where I left the alias because of the comment:

        /// <summary>
        /// This is a set of wildcard chars which can cause a file extension to be invalid 
        /// </summary>
        private static readonly char[] s_wildcards = MSBuildConstants.WildcardChars;

Example from ToolsetConfigurationREader.cs:

        private static readonly char[] s_separatorForExtensionsPathSearchPaths = MSBuildConstants.SemicolonChar;

Reply:
IIRC I agree with all of your decisions on that axis 👍